.gitignore

```diff
@@ -33,3 +33,4 @@ local/config/vufind/*.ini
 local/config/vufind/*.json
 local/config/vufind/*.key
 local/config/vufind/*.yaml
+*.DS_Store
```

Comment:
If this is an intentional addition, maybe it's worth submitting as a separate PR, since it's a general change unrelated to the rest of this PR.